api/invite.go

```diff
@@ -937,6 +936,20 @@ func (a *Api) SendInvite(res http.ResponseWriter, req *http.Request, vars map[st
 			log.Printf("SendInvite: invited [%s] user already has or had an invite", ib.Email)
 			return
 		} else {
+
+			if invitedUsr == nil {
+				// we return an error as the invitedUser does not exist yet
+				statusErr := &status.StatusError{Status: status.NewStatus(http.StatusForbidden, STATUS_ERR_FINDING_USER)}
+				a.sendModelAsResWithStatus(res, statusErr, statusErr.Code)
+				return
+			}
+
+			if invitedUsr.HasRole("patient") {
```

Comment:
```suggestion
			if invitedUsr != nil && invitedUsr.HasRole("patient") {
```
Replace the test block from above with this